### PR TITLE
Sliders, boxes and googles

### PR DESCRIPTION
**Pull request by Stanton**
An assortment of things required for new Continuum CMS features which improve the initial installation and setup of the software.

Addresses #158 

# Slider

Intended to be used on initial CMS setup, has a familial relationship to the Continuum login screen (so would appear on the stars background). Also close to some CXM branded sign-in prototypes that @jamesjacobs did some time ago.

screen shot 2016-09-09 at 21 44 27

# Google result

Used to show how certain content and metadata would appear in a search result. Uses exact sizes, spacings and colours from Google's stylesheet.

screen shot 2016-09-09 at 21 47 44

# Box

Shown here as the grey box containing the google result, used to increase contrast of an element without using a panel, which are too dark for cases like this.

screen shot 2016-09-09 at 21 47 44

```html
<div class="google-result box box--rounded">
    <span class="google-result__link">Continuum City Council</span>
    <cite class="google-result__cite">www.cmsdomain.com/</cite>
    <span class="google-result__description">Site description</span>
</div>
```

### CODE REVIEW

File: stylesheets/_component.slider.scss [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+.slider-title {
+    font-size: 1.2em;
+    line-height: 1.2;
+}
+
+// form alignment
+.slider {
+    .legend {
+        line-height: 1em;
```
Comment (langy) [inline stylesheets/_component.slider.scss]: Unit-based line height intentional?

File: stylesheets/_component.slider.scss [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+.slider-logo {
+    float: left;
+    margin: 0 10px 10px 0;
+    width: 45px;
+}
+
+.slider-title {
+    font-size: 1.2em;
+    line-height: 1.2;
```
Comment (langy) [inline stylesheets/_component.slider.scss]: Are there not any vars we could use here?

File: stylesheets/_component.slider.scss [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+// form alignment
+.slider {
+    .legend {
+        line-height: 1em;
+        margin-bottom: 5px;
+        border-bottom: 1px solid #ccc;
```
Comment (langy) [inline stylesheets/_component.slider.scss]: Hard-coded colour on purpose?

File: stylesheets/_component.slider.scss [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        margin-bottom: 5px;
+    }
+
+    .form__group--top .control__label {
+        margin-bottom: 0;
+    }
+
+    .form__control {
+        height: 32px;
+        line-height: 32px;
```
Comment (langy) [inline stylesheets/_component.slider.scss]: No suitable vars available for these?

**Comment by langy** [comment]
Apart from comments, 👍

File: stylesheets/_component.slider.scss (hunk 3 above)
Comment (Stanton) [inline stylesheets/_component.slider.scss]: built quite a while ago, will change to colour vars

**Pull request merged**